tests/test_api/test_endpoints.py

```diff
@@ -236,12 +236,13 @@ def test_submit_changesheet():
     assert True
 
 
+# THIS TEST IS SKIPPED - FOR NOW JUST UPDATE NAMING
 @pytest.mark.skip(
-    reason="Skipping because race condition causes  http://fastapi:8000/nmdcschema/ids/nmdc:wfrqc-11-t0tvnp52.2 to 404?"
+    reason="Skipping because race condition causes http://fastapi:8000/nmdcschema/ids/nmdc:wfrqc-11-t0tvnp52.2 to 404?"
 )
-def test_submit_workflow_activities(api_site_client):
+def test_submit_workflow_executions(api_site_client):
     test_collection, test_id = (
-        "read_qc_analysis_activity_set",
+        "read_qc_analysis_activity_set",  # does this exist in Berkeley schema?
```

Comment:
looks like no. From <https://microbiomedata.github.io/berkeley-schema-fy24/Database/>, I think there may only be [workflow_execution_set](https://microbiomedata.github.io/berkeley-schema-fy24/workflow_execution_set/).

Reply:
I agree with Donny. Also im curious why this test passes with this no longer existing collection. 
From this PR:
 tests/test_api/test_endpoints.py ss.....s.                               [ 25%]

Reply:
oh nevermind, i see some other comments about this being skipped.